app/image/suite_test.go

```diff
@@ -0,0 +1,35 @@
+package image
```

Comment:
Oh okay then for some reason github collapsed the changes to image_test.go and I missed them.